packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -461,167 +460,28 @@ export class Relayer {
     return RelaySubmitType.Ignore;
   }
 
-  private async slowRelay(deposit: Deposit, realizedLpFeePct: BN) {
```

Comment:
great thing about this PR is we can remove all this custom tx processing logic for each transaction we can send!